.travis.yml

```diff
@@ -16,7 +16,7 @@ addons:
 
 env:
   global:
-    - ANT_HOME=/usr/share/ant
+    - ANT_HOME=$HOME/apache-ant-1.10.2
```

Comment:
This is inconsistent with M2_HOME: 1st points to $HOME/..., 2nd to /usr/local/...
I would expect those utilities with the same prefix. But I did not see how it looks in Travis CI image so if it works fine ....it should be ok.

Reply:
build uses default version of Maven (the one from `/usr/local/...`) and custom version of ant (explicitly downloaded and unzipped to`$HOME/...`); ant in `/usr/local/...` is version 1.9.6, or sth like that, and newer version is not available in trusty repo, so it can't be updated through `apt` (or at least not using default repos)